lib/utils.js

```diff
@@ -81,6 +87,34 @@ function formatTime (epoch, translateTime = false) {
   return dateformat(instant, `UTC:${translateTime}`)
 }
 
+/**
+ * Replacement for the Date() constuctor. The argument may be any argument
+ * that is valid for the Date constuctor, or an epoch provided as a string.
+ *
+ * @epoch {string|number} - Argument for the date constructor.
+ *
+ * @returns {Date} date
+ */
+function createDate (epoch) {
```

Comment:
This function should be added to `module.exports.internals` and tested explicitly in `test/lib/utils.internals.test.js`.